sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Artifact/RegistryArtifact.cs

```diff
@@ -9,7 +9,10 @@
 
 namespace Azure.Containers.ContainerRegistry
 {
-    /// <summary> A helper class that groups information and operations about an image or artifact in this container registry. </summary>
+    /// <summary> `Artifact` is the general term for items stored in a container registry,
+    /// and can include Docker images or other Open Container Initiative (OCI) artifact types.
+    ///
```

Comment:
I'm not sure if the spacing translates to the actual docs or not - you may need to do this with the `<para>` tag if you want that kind of formatting.